docs/modules/ROOT/pages/performance_analysis/dpmAcaGrpcTest.adoc

```diff
@@ -0,0 +1,215 @@
+
+image::p1.png["Test Deployment",width=488,height=302]
+
+[arabic, start=3]
+. *Test results*
+
+[cols=",,,,,,,,",options="header",]
+|===
+|*Log Time Cost Summary (ms)* | | | | | | | |
+|*Thread* |*1 -1^st^* |*1-2^nd^* |*32 -1^st^* |*32-2^nd^* |*56 -1^st^* |*56-2^nd^* |*128 -1^st^* |*128-2^nd^*
+|*Average* |11.14 |10.37 |16.50 |14.40 |26.54 |25.60 |70.64 |61.54
+|*MIN* |9 |9 |10 |9 |10 |9 |10 |9
+|*MAX* |48 |18 |223 |116 |364 |160 |603 |352
+|*MEDIAN* |11 |10 |11 |11 |11 |11 |11 |11
+|*>AVERAGE* |28.88% |40.35% |21.48% |20.09% |21.80% |20.09% |21.03% |20.00%
+|*<AVERAGE* |71.12% |59.65% |78.52% |79.91% |78.20% |79.91% |78.97% |80.00%
+|*99% TILE* |13 |12 |67 |43 |107.04 |116.03 |331 |312
+|*95% TILE* |12 |12 |36 |31 |83 |89 |305 |277
+|*90% TILE* |12 |11 |32 |28 |78 |84 |292 |262
+|===  
+
+
+[arabic, start=4]
+. *Test results analysis*
+[loweralpha]
+.. {blank}
++
+____
+*System Resource Usage on F-- ALCOR DATAPLANE MANAGER (context switch)*
+____
+
+____
+image::p2.png["context swith Alcor DataPlane Manager",width=553,height=302]
+____
+
+[loweralpha, start=2]
+. {blank}
++
+____
+*Time Cost on F -- ALCOR DATAPLANE MANAGER round trip*
+____
+
+____
+image::omax.png["thread number of Alcor DataPlane Manager",width=553,height=302]
+____
+
+____
+*Time Cost on single execution max -- ALCOR DATAPLANE MANAGER round trip*
+____
+
+____
+image::jmax.png["thread number of single execution max time Alcor DataPlane Manager",width=553,height=302]
+____
+
+[loweralpha, start=3]
+. {blank}
++
+____
+*Time Cost Charts for round trip when thread number change on F*
+____
+
+single thread
+____
+image::1-1.png["1 thread",width=276,height=165]
+____
+
+single thread -- 2nd time
+____
+image::1-2.png["1 thread 2nd time",width=276,height=165]
+____
+
+32 threads
+____
+image::32-1.png["32 thread",width=262,height=156]
+____
+
+32 threads -- 2nd time
+____
+image::32-2.png["32 thread 2nd time",width=262,height=156]
+____
+
+56 threads
+____
+image::56-1.png["56 thread",width=262,height=156]
+____
+
+56 threads -- 2nd time
+____
+image::56-2.png["56 thread 2nd time",width=262,height=156]
+____
+
+128 threads
+____
+image::128-1.png["128 thread",width=262,height=156]
+____
+
+128 threads  -- 2nd time
+____
+image::128-2.png["128 thread 2nd time",width=262,height=156]
+____
+
+for 256 threads and below, the success rate is 100%
+
+
+for 512 threads above, the success rate is under 100%
+
+____
+* 10k neighbor, overall time cost for different concurrent thread number*
+____
+
+____
+image::1w-ov.png["10k neighbor, overall time cost for different concurrent thread number",width=553,height=302]
+____
+
+____
+* 10k neighbor, every connection time cost for different concurrent thread number*
+____
+
+____
+image::1w-ov-jc.png["10k neighbor, every connection time cost for different concurrent thread number",width=553,height=302]
+____
+
+
+____
+* when neighbor number changed, every connection time cost and overall time cost for different concurrent thread number*
```

Comment:
This image is important. Let us work to collect more data based on two dimensions (concurrent thread # and neighbor numbers), fix one and adjust the other.